src/squidpy/gr/_utils.py

```diff
@@ -19,6 +20,14 @@
 from squidpy._docs import d
 from squidpy._utils import NDArrayA, _unique_order_preserving
 
+CAN_USE_SPARSE_ARRAY = Version(anndata.__version__) >= Version("0.11.0")
+if not CAN_USE_SPARSE_ARRAY:
+    from anndata._core.views import ArrayView
+    from anndata._core.views import SparseCSCMatrixView as SparseCSCView
+    from anndata._core.views import SparseCSRMatrixView as SparseCSRView
+else:
+    from anndata._core.views import ArrayView, SparseCSCView, SparseCSRView
```

Comment:
This is a ping for @timtreis and @giovp that sparse _array_ support is coming: https://docs.scipy.org/doc/scipy/reference/sparse.html

Reply:
See the note